hack/setup/infra/manage.wva-helm.sh

```
@@ -0,0 +1,110 @@
+#
+# Environment variables for custom Helm values:
+#   WVA_EXTRA_ARGS - Additional helm install arguments
+#   WVA_PROMETHEUS_URL - Prometheus URL for WVA (default: http://prometheus-kube-prometheus-prometheus.monitoring:9090)
+#
+# Installs the WVA controller from the official llm-d OCI registry.
+# WVA watches VariantAutoscaling CRs and emits wva_desired_replicas metrics.
+
+# INIT
+SCRIPT_DIR="$(cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" && pwd)"
+
+source "${SCRIPT_DIR}/../common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../common.sh was not specified as input (see shellcheck -x).

(SC1091)